src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -1045,7 +1045,9 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 			this._viewsService.openView(Markers.MARKERS_VIEW_ID);
 		} else if (task.command.presentation && (task.command.presentation.focus || task.command.presentation.reveal === RevealKind.Always)) {
 			this._terminalService.setActiveInstance(terminal);
-			this._terminalGroupService.showPanel(task.command.presentation.focus);
+			if (terminal.target !== TerminalLocation.Editor) {
```

Comment:
let's call this instead

https://github.com/microsoft/vscode/blob/95f948717b45f8ee70672092bd93953cb2551744/src/vs/workbench/contrib/terminal/browser/terminalService.ts#L556-L566

Reply:
I see.